server/src/main/java/org/apache/druid/metadata/SQLMetadataSegmentManager.java

```diff
@@ -106,6 +106,8 @@
   private final SQLMetadataConnector connector;
 
   // Volatile since this reference is reassigned in "poll" and then read from in other threads.
```

Comment:
@surekhasaharan to the list of things to address in a follow-up PR: make the comment to a field a Javadoc comment and make all references to methods and fields that appear in it links like: `{@link }`.